contexts/cloud-messaging.provider.web.tsx

```
@@ -0,0 +1,47 @@
+import {
+  useContext,
+  createContext,
+  type PropsWithChildren,
+  useEffect,
+} from "react";
+
+import { getToken } from "firebase/messaging";
+import { messaging } from "@/utils/messaging-web";
+
+interface CloudMessagingContextProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)